sdk/flutter/lib/picovoice.dart

```diff
@@ -10,42 +10,68 @@
 //
 
 import 'picovoice_error.dart';
-import 'package:porcupine/porcupine.dart';
-import 'package:porcupine/porcupine_error.dart' as porcupineErr;
-import 'package:rhino/rhino.dart';
-import 'package:rhino/rhino_error.dart' as rhinoErr;
+import 'package:porcupine_flutter/porcupine.dart';
+import 'package:porcupine_flutter/porcupine_error.dart';
+import 'package:rhino_flutter/rhino.dart';
+import 'package:rhino_flutter/rhino_error.dart';
+
+typedef WakeWordCallback = Function();
+typedef InferenceCallback = Function(PicovoiceInference inference);
+
+class PicovoiceInference {
```

Comment:
Can't we just use RhinoInference here? Why do we need to create a new class that duplicates it?